### DISCUSSION TITLE
关于yande的下载相关想法

### DISCUSSION THREAD

**Discussion by KanoueShinku** [drunkg00se/Pixiv-Downloader#24] [Ideas]
能否在下载设置文件重命名那里多加一个{desc}/{tag}？想把图片的tag也附带上方便以后文件检索

**Comment by drunkg00se** [comment]
> 能否在下载设置文件重命名那里多加一个{desc}/{tag}？想把图片的tag也附带上方便以后文件检索

可参考greasyfork上的[讨论](https://greasyfork.org/zh-CN/scripts/432150-pixiv-downloader/discussions/268933)
